addons/stock_account/models/product.py

```diff
@@ -739,9 +743,7 @@ class ProductCategory(models.Model):
         help="""Standard Price: The products are valued at their standard cost defined on the product.
         Average Cost (AVCO): The products are valued at weighted average cost.
         First In First Out (FIFO): The products are valued supposing those that enter the company first will also leave it first.
-        """,
-        tracking=True,
```

Comment:
Is there a reason for this change? Note that I didn't look through the other comments people made :sloth:

Reply:
this attribute doesn't seem to exist anymore, I am personally fine to use the opportunity to get rid of it

Reply:
> this attribute doesn't seem to exist anymore, I am personally fine to use the opportunity to get rid of it

Huh? :confused: it does still exist in the master runbot or am I missing something?
![image](https://github.com/odoo/odoo/assets/57942143/5093a0e6-9bab-40a1-b7a8-96fafd641bc5)